sdk/datafactory/Azure.ResourceManager.DataFactory/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed an issue that exception throws when `Uri` type field is empty during deserialization of `SelfHostedIntegrationRuntimeStatus`.
+    - Change the name of ServiceUris to ServiceUriStringList and change the type it contains from Uri to string
```

Comment:
Remove this line since the previous item includes the point already.

Reply:
fixed